File: TransCoder_tokenized_test_set_functions/cpp/CHECK_ARRAY_CONTAINS_CONTIGUOUS_INTEGERS_DUPLICATES_ALLOWED_1.cpp
bool areElementsContiguous ( int arr [ ], int n ) {
  unordered_set < int > us;
  for ( int i = 0;
  i < n;
  i ++ ) us . insert ( arr [ i ] );
  int count = 1;
  int curr_ele = arr [ 0 ] - 1;
  while ( us . find ( curr_ele ) != us . end ( ) ) {
    count ++;
    curr_ele --;
  }
  curr_ele = arr [ 0 ] + 1;
  while ( us . find ( curr_ele ) != us . end ( ) ) {
    count ++;
    curr_ele ++;
  }
  return ( count == ( int ) ( us . size ( ) ) );
}

File: TransCoder_tokenized_test_set_functions/python/CHECK_ARRAY_CONTAINS_CONTIGUOUS_INTEGERS_DUPLICATES_ALLOWED_1.py
def areElementsContiguous ( arr ) :
    us = set ( )
    for i in arr : us . add ( i )
    count = 1
    curr_ele = arr [ 0 ] - 1
    while curr_ele in us :
        count += 1
        curr_ele -= 1
    curr_ele = arr [ 0 ] + 1
    while curr_ele in us :
        count += 1
        curr_ele += 1
    return ( count == len ( us ) )
